projects/addon-mobile/components/sheet/components/sheet-bar/sheet-bar.component.ts

```diff
@@ -13,16 +12,15 @@ import {TUI_SHEET, TUI_SHEET_SCROLL} from '../../sheet-tokens';
     changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class TuiSheetBarComponent {
+    private readonly sheet = inject(TUI_SHEET);
+    // TODO: why TUI_SHEET_SCROLL returned number?
+    private readonly scroll$ = inject<Observable<number>>(TUI_SHEET_SCROLL);
```

Comment:
Should we just fix the type of injection token ?

https://github.com/taiga-family/taiga-ui/blob/7b824fe415f732f865895c94632a0ace9d11d485/projects/addon-mobile/components/sheet/sheet-tokens.ts#L13